src/ploneorg/addonlisting/utils.py

```diff
@@ -118,25 +131,42 @@ def update_addon(context, request=None):
             api.portal.show_message("Add'on %s has been updated" %
                                     (addon.title),
                                     request=request, type='info')
-            log.info(u'Finished to update: %s', addon.title)
+            if verbose:
+                log.info(u'Finished to update: %s', addon.title)
         else:
-            log.info(u'something went wrong on update %s', addon.title)
+            if verbose:
+                log.info(u'something went wrong on update %s', addon.title)
 
         if request is not None:
             request.response.redirect(context.absolute_url())
 
 
-def update_addons(context, request=None):
+def update_addons(context, request=None, verbose=False, limit=0):
     addon_folder = context
 
-    # get Add'on List
-    addons = api.content.find(context=addon_folder, portal_type="AddOn")
+    if limit:
```

Comment:
that looks a bit wired. especially if that parameter sort_limit exists, it is more to find if  there is a special input for no limit.